app/components/AppearanceSettings.vue.ts

```diff
@@ -2,21 +2,26 @@ import Vue from 'vue';
 import { Component } from 'vue-property-decorator';
 import { Inject } from '../util/injector';
 import GenericForm from 'components/obs/inputs/GenericForm.vue';
+import { $t } from 'services/i18n';
 import { TObsFormData } from 'components/obs/inputs/ObsInput';
 import { ICustomizationServiceApi, ICustomizationSettings } from 'services/customization';
+import { WindowsService } from 'services/windows';
 
 @Component({
   components: { GenericForm }
 })
 export default class AppearanceSettings extends Vue {
 
   @Inject() private customizationService: ICustomizationServiceApi;
+  @Inject() private windowsService: WindowsService;
 
   settingsFormData: TObsFormData = null;
+  enableFFZEmotes: Boolean = false;
```

Comment:
The type you want here is `boolean` (lowercase "b").  You can read more about the difference here: https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html

Also, since you're immediately assigning a boolean value, you can technically omit the type annotation entirely if you want and TypeScript will figure it out.